dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,139 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
+CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
+SOURCE_DIR="pandas-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	pandas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	setuptools_python
+	setuptools_python3
+	cython_python
+	cython_python3
```

Comment:
Oops, I missed this one.
```diff
BUILD_REQUIRES="
	haiku${secondaryArchSuffix}_devel
-	setuptools_python
-	setuptools_python3
-	cython_python
-	cython_python3
-	python_numpy
-	python3_numpy
-	dateutil_python
-	dateutil_python3
-	pytz_python
-	pytz_python3
 	"
```
(These dependencies are already mentioned later, and should not appear here.)